src/core/analytics.js

```diff
@@ -195,6 +196,17 @@ class Analytics {
     );
   }
 
+  validateConfig(requiredConfigsList, configsFromDashboard, intg) {
+    if(!requiredConfigsList){
+      throw Error(`${intg}: required configs not present`);
+    }
+    requiredConfigsList.forEach(config => {
+      const configVal = configsFromDashboard[config];
+      if (!isDefinedAndNotNull(configVal) || (typeof configVal === "string" && configVal.length === 0))
+        throw Error(`${intg}: ${config} is either not present or does not have valid value`);
```

Comment:
The error message needs to be more helpful to the user.

```suggestion
        throw Error(`${intg}: The destination configuration parameter "${config}" is either unavailable or invalid. Please check your destination settings in the dashboard.`);
```